metadata-io/src/main/java/com/linkedin/metadata/entity/ebean/EbeanAspectDao.java

```diff
@@ -233,15 +232,13 @@ public EntityAspect getAspect(@Nonnull final EntityAspectIdentifier key) {
   }
 
   @Override
-  @Nullable
```

Comment:
This also seems like just style changes? Removing these annotations doesn't do much right?

Reply:
I can revert this, but these annotations don't make any sense on primitive return methods. One is an int and one is a void

Reply:
True